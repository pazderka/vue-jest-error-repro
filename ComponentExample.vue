<template>
  <div class="hello">Hello world.</div>
</template>

<script>
export default {
  name: 'ComponentExample',
}
</script>

<style>
.hello {
  color: #000;
}
</style>
