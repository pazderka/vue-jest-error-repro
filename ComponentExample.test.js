import { shallowMount } from '@vue/test-utils'
import ComponentExample from './ComponentExample.vue'

describe('Component', () => {

    test('Matches snapshot', () => {
        const wrapper = shallowMount(ComponentExample)
        expect(wrapper.element).toMatchSnapshot()
    })

})
