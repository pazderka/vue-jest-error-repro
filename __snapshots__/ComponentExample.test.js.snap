// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Matches snapshot 1`] = `
<div
  class="hello"
>
  Hello world.
</div>
`;
